backend/CreateUserLambda/App.config

```diff
@@ -0,0 +1,10 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+
+<configuration>
+  <configSections>
+    <section name="aws" type="Amazon.AWSSection, AWSSDK.Core"/>
+  </configSections>
+  <aws region="us-west-2">
```

Comment:
Why is the region set to us-west?

Reply:
Oversight when creating a standard project through Visual Studio plugins. 

I do not believe this file is used, as I do not deploy through VS